Makefile

```diff
@@ -11,7 +11,6 @@ spec.html: spec.bs
 	                       --output spec.html \
 	                       --write-out "%{http_code}" \
 	                       --header "Accept: text/plain, text/html" \
-	                       -F die-on=warning \
```

Comment:
Does this make the build less permissive?
Could you mention this change in the commit description?

Reply:
Removing the die-on=warning makes it more permissive.  In my local tests, I was seeing it die with a warning message that I didn't understand because the spec is otherwise processed correctly.  The errors and warnings only come out one at a time, and the messages are not at all clear, which is very tedious to deal with.  Need to find a better error reporting process.  I'll add to the description above.

Reply:
Can we add some context to the commit message?  like instead of "I removed the die-on=warning" say "Ignore warnings during build"

Reply:
Done.  But this is probably a temporary workaround anyway.  Apparently, building locally will display more useful warnings.